purchases/src/main/kotlin/com/revenuecat/purchases/SubscriptionInfo.kt

```diff
@@ -74,6 +76,22 @@ class SubscriptionInfo(
      * The transaction id in the store of the subscription.
      */
     val storeTransactionId: String?,
+    /**
+     * Date when the subscription will auto-resume.
```

Comment:
Maybe add more details that it will only have a value if the subscription is paused and that it's for Google subs only (I think?)